src/pages/home-page-slides/slides-page/slides-page.js

```diff
@@ -27,9 +28,13 @@ const { mainPageTitle, slideTitle } = config.titles.homePageSliderTitle;
 const { pathToAddHomePageSlide } = config.routes;
 
 const SlidesPage = () => {
-  const { openSuccessSnackbar } = useSuccessSnackbar();
   const styles = useStyles();
+  const common = useCommonStyles();
```

Comment:
Can you rename this `common` variable to styles or classes?

Reply:
fixed